platform/security/handler/security-handler-pki/src/main/java/org/codice/ddf/security/handler/pki/CrlChecker.java

```diff
@@ -50,16 +50,13 @@
     private static String encryptionPropertiesLocation = new AbsolutePathResolver(
             "etc/ws-security/server/encryption.properties").getPath();
 
-    private AtomicReference<CRL> crlCache = new AtomicReference<>();
+    private static AtomicReference<CRL> crlCache = new AtomicReference<>();
 
-    private CrlRefresh refresh = new CrlRefresh();
+    private static final CrlRefresh REFRESH = new CrlRefresh();
 
-    /**
-     * Constructor method. Reads encryption.properties and sets CRL location
-     */
-    public CrlChecker() {
+    static {
```

Comment:
❓ Should we move this in the `CrlRefresh` constructor instead of having a `static` block here?